README.md

```diff
@@ -1,16 +1,248 @@
-# Title
+# Methods in Ruby
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Describe routines and procedures for our code using methods
+- Define a method
+- Recognize how to call methods
 
 ## Introduction
 
-## SWBAT 1
+A method in Ruby is a _block_ of code that returns a value. With methods, we can
+organize our code into chunks of reusable functionality that can be easily used
+by other areas of their program.
 
-## SWBAT 2
+Ruby methods are very similar to functions in any other programming language.
+Ruby methods are used to bundle one or more repeatable statements into a single
+unit.
+
+In this lesson we'll introduce methods, distinguish them from data types, and
+cover how to create and execute them in your Ruby program.
+
```

Comment:
These paragraphs don't feel..united to me. I think they can be tighter....

e.g.


_Methods_ are used to bundle one or more activities into a single unit. In daily life we do this all the time: "get ready for work" means: "take a shower, walk the dog, eat breakfast." But each of _those_ activities are made up of other sub-activities, and sub-sub activities.  Take a shower involves steps like "wash hair" which has sub-steps like "wet head under shower, apply shampoo, etc."

Nearly all programming languages have the idea of "bundling up work" under a programmer-created name. While different languages might call them "subroutines," "methods," or "functions," they all mean the same thing: grouping work.

In this lesson...